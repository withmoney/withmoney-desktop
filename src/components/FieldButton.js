import React from 'react';

const FieldButton = ({ children, ...othersProps }) => (
  <div className="field">
    <button
      type="button"
      className="button"
      {...othersProps}
    >
      {children}
    </button>
  </div>
);


export default FieldButton;
