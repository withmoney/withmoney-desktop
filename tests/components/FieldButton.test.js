import renderer from 'react-test-renderer';
import FieldButton from 'components/FieldButton';

describe('FieldButton', () => {
  let Component;

  beforeAll(() => {
    Component = global.setup(FieldButton);
  });

  it('should render a component correctly', () => {
    const wrapper = renderer.create(
      Component({
        children: 'Text',
      }),
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
