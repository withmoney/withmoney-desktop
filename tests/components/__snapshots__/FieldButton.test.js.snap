// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldButton should render a component correctly 1`] = `
<div
  className="field"
>
  <button
    className="button"
    type="button"
  >
    Text
  </button>
</div>
`;
